SDWebImage/Core/SDAnimatedImagePlayer.m

```diff
@@ -13,6 +13,7 @@
 #import "SDInternalMacros.h"
 
 @interface SDAnimatedImagePlayer () {
+    SD_LOCK_DECLARE(_lock);
     NSRunLoopMode _runLoopMode;
```

Comment:
From the usage habit, I prefer this statement: `SD_LOCK_DECLARE  _lock`

Reply:
Not possible for me...Can you have a try?

You should write a macro which expand to the following result: 

```objective-c
os_unfair_lock _lock API_AVAILABLE(ios(10.0), tvos(10), watchos(3), macos(10.12));
OSSpinLock _lock;
```

Reply:
There are 2 variable, not single one.

Reply:
I know there are must using `SD_LOCK_DECLARE(_lock)` from you case.